### PR TITLE
Tweak, refine, and update CoC language

### PR DESCRIPTION
**Pull request by mjgiarlo**
Aiming for increased consistency, even gooderer grammar, and cleaner disambiguation / more explicit call-outs regarding discriminatory, sexual, and sexualized language and images.

NOTE: I'm reading this as a 2021 CSV, and I figured I might as well toss some edits in here while I'm reading it. I'm not wedded to any particular change, and am happy to work with the community to get the palatable deltas merged. Thanks for considering!

### CODE REVIEW

**Comment by mjgiarlo** [comment]
Per the https://github.com/code4lib/code-of-conduct/blob/main/how_changes_are_made.md document, I am pinging the CSS to review and comment on these changes: @aaron-collier @bobbi-SMR @kayiwa @anarchivist @aslaughter

File: code_of_conduct.md [hunk 1]
```diff
@@ -6,122 +6,123 @@ CodeOfConduct4Lib
...
 1. Initial Incident
 
     If you are being harassed, notice that someone else is being harassed,
     or have any other concerns, __and you feel comfortable speaking with
-    the offender__, please inform the offender that he/she/ze has affected you 
-    negatively. Oftentimes, the offending behavior is unintentional, and the
-    accidental offender and offended will resolve the incident by having
-    that initial discussion.
+    the offender__, please inform the offender that they have affected you 
+    negatively. The offending behavior may be unintentional, and the
```
Comment (jeremyf) [inline code_of_conduct.md:32]: I like this. Removes some wiggle and apologizing language

**Review by jeremyf — APPROVED** [review]

File: code_of_conduct.md [hunk 2]
```diff
@@ -6,122 +6,123 @@ CodeOfConduct4Lib
 Short link: <http://bit.ly/coc4lib>
 
 Code4Lib seeks to provide a welcoming, professionally engaging, fun, and
-safe conference experience and ongoing community for everyone. We do not
-tolerate harassment in any form. Discriminatory language and imagery
-(including sexual) is not appropriate for any event venue, including talks,
-or any community channel such as the chatroom or mailing list.
+safe conference (and ongoing community) experience for everyone. We do not
```
Comment (mjgiarlo) [inline code_of_conduct.md:9]: Use parentheses to clearly delineate that the CoC is in force in two separate but related contexts.

File: code_of_conduct.md [hunk 3]
```diff
@@ -6,122 +6,123 @@ CodeOfConduct4Lib
...
+tolerate harassment in any form. Discriminatory language and imagery,
+including sexual or sexualized language and imagery, is not appropriate
```
Comment (mjgiarlo) [inline code_of_conduct.md:11]: I found the prior trailing parenthetical `(including sexual)` awkward and unclear: was it modifying imagery only? Or is sexual language a form of discriminatory language? 

Call out sexual or sexualized language and imagery in particular, instead, to disambiguate.

File: code_of_conduct.md [hunk 4]
```diff
@@ -6,122 +6,123 @@ CodeOfConduct4Lib
...
-body size, race, age, religious or political beliefs; sexual or
-discriminatory images in public spaces (including online); deliberate
-intimidation, stalking, following, harassing photography or recording;
-sustained disruption of talks or other events, inappropriate physical
-contact, and unwelcome sexual attention.
+person or group, or that produces an unsafe environment.  It includes 
+offensive verbal comments or non-verbal expressions related to gender, gender
+identity, gender expression, sexual orientation, disability, physical 
+appearance, body size, race, age, religious or political beliefs; sexual, 
+sexualized, or discriminatory images in public (including online) spaces;
```
Comment (mjgiarlo) [inline code_of_conduct.md:20]: Adding the notion of sexualized here to call that out explicitly.

File: code_of_conduct.md [hunk 5]
```diff
@@ -6,122 +6,123 @@ CodeOfConduct4Lib
...
-discriminatory images in public spaces (including online); deliberate
-intimidation, stalking, following, harassing photography or recording;
-sustained disruption of talks or other events, inappropriate physical
-contact, and unwelcome sexual attention.
+person or group, or that produces an unsafe environment.  It includes 
+offensive verbal comments or non-verbal expressions related to gender, gender
+identity, gender expression, sexual orientation, disability, physical 
+appearance, body size, race, age, religious or political beliefs; sexual, 
+sexualized, or discriminatory images in public (including online) spaces; 
+deliberate intimidation, stalking, or following; harassing photography or
```
Comment (mjgiarlo) [inline code_of_conduct.md:21]: Use semi-colons consistently to make the list of harassment vectors clearer.

File: code_of_conduct.md [hunk 6]
```diff
@@ -6,122 +6,123 @@ CodeOfConduct4Lib
...
-    negatively. Oftentimes, the offending behavior is unintentional, and the
-    accidental offender and offended will resolve the incident by having
-    that initial discussion.
+    the offender__, please inform the offender that they have affected you 
+    negatively. The offending behavior may be unintentional, and the
+    offender and offended may resolve the incident by having that initial
+    discussion themselves.
 
-    Code4Lib recognizes that there are many reasons speaking directly to
+    Code4Lib understands that there are many reasons speaking directly to
```
Comment (mjgiarlo) [inline code_of_conduct.md:36]: Using `recognizes` here strikes me as semi-formal, which could unintentionally communicate to the reader that Code4Lib has a preference that the offended work things out directly with the offender. Soften the verb to something that conveys a touch more empathy to head that off.

File: code_of_conduct.md [hunk 7]
```diff
@@ -6,122 +6,123 @@ CodeOfConduct4Lib
...
     unfamiliarity with the conference or its participants, [lack of spoons](https://web.archive.org/web/20191117210039/https://butyoudontlooksick.com/articles/written-by-christine/the-spoon-theory/),
-    and concerns for personal safety). If you don't feel comfortable
+    or concerns for personal safety). If you do not feel comfortable
     speaking directly with the offender for any reason, skip straight to
     step 2.
 
 2. Escalation
 
-    If the offender insists that he/she/ze did not offend, if offender is
+    If the offender insists that they did not offend, if the offender is
```
Comment (mjgiarlo) [inline code_of_conduct.md:45]: Preferring `they` here because simpler and broader.

File: code_of_conduct.md [hunk 8]
```diff
@@ -6,122 +6,123 @@ CodeOfConduct4Lib
...
-    If you are in the #code4lib IRC, the zoia command to list people
-    designated as channel helpers is @helpers . There is at least one helper
+    If you are in the #code4lib IRC channel, the zoia command to list people
+    designated as channel helpers is `@helpers`. There is at least one helper
     in the channel at most times.
 
-    If you are in the code4lib Slack, contact one of the admins. You can find this list by clicking the the "⋮" button
-    in the top right and then selecting "Workspace Directory." You can filter the list to show only Admins by using the
-    select list.
+    If you are in the code4lib Slack, contact one of the admins. You can find
```
Comment (mjgiarlo) [inline code_of_conduct.md:59]: Whitespace changes only in this para and the next.

File: code_of_conduct.md [hunk 9]
```diff
@@ -6,122 +6,123 @@ CodeOfConduct4Lib
...
+    In addition, there are [Community Support Volunteers](css_volunteers.md) whom you 
+    are welcome to involve.
 
 3. Wider community response to Incident:
 
     If the incident doesn't pass the first step (discussion reveals offense
-    was unintentional, apologies said, public note or community is informed
-    of resolution), then there's not much the community can do at this point
-    since the incident was resolved without outside intervention.
+    was unintentional, apologies offered and accepted, public note or
```
Comment (mjgiarlo) [inline code_of_conduct.md:72]: Apology should be *accepted*, not merely *said*.

File: code_of_conduct.md [hunk 10]
```diff
@@ -6,122 +6,123 @@ CodeOfConduct4Lib
...
+    intervention.
 
     If incident results in corrective action, the community should support
     the decision made by the Help in Step 2 if they choose corrective action,
     like ending a talk early or banning from the listserv, as well as
     support those harmed by the incident, either publicly or privately
     (whatever individuals are comfortable with).
 
-    If the Help in Step 2 run into issues implementing the CoC, then the
+    If the Help in Step 2 runs into issues implementing the CoC, then the
```
Comment (mjgiarlo) [inline code_of_conduct.md:83]: Grammar.

File: code_of_conduct.md [hunk 11]
```diff
@@ -6,122 +6,123 @@ CodeOfConduct4Lib
...
     should revise the CoC as they see fit.
 
-    In Real Life people will have opinions about how the CoC is enforced.
+    In Real Life, people will have opinions about how the CoC is enforced.
     People will argue that a particular decision was unfair, and others will
-    say that it didn't go far enough. We can't stop people having
-    opinions, but what we could do here is have constructive discussions
-    that lead to something tangible (affirmation of decision, change in CoC,
-    modify decision, etc,).
+    say that it didn't go far enough. We wouldn't dream of asking people to
```
Comment (mjgiarlo) [inline code_of_conduct.md:89]: No, we can't stop people having opinions and furthermore, IMO, we don't even want to suggest that's a thing we'd support.

File: code_of_conduct.md [hunk 12]
```diff
@@ -6,122 +6,123 @@ CodeOfConduct4Lib
...
 
 ## Sanctions
 
 Participants asked to stop any harassing behavior are expected to comply
-immediately.  If a participant engages in harassing behavior, organizers may
+immediately. If a participant engages in harassing behavior, organizers may
 take any action they deem appropriate, including warning the offender,
-expulsion from the Code4Lib event, or banning the offender from a chatroom
-or mailing list.
+expelling the offender from the Code4Lib event, or banning the offender from
```
Comment (mjgiarlo) [inline code_of_conduct.md:99]: Parallel construction.

File: code_of_conduct.md [hunk 13]
```diff
@@ -6,122 +6,123 @@ CodeOfConduct4Lib
...
+a chatroom, Slack workspace, or mailing list.
```
Comment (mjgiarlo) [inline code_of_conduct.md:100]: Include Slack.

File: code_of_conduct.md [hunk 14]
```diff
@@ -6,122 +6,123 @@ CodeOfConduct4Lib
...
 
-* warning the harasser to cease their behavior and that any further reports
-will result in other sanctions
-* requiring that the harasser avoid any interaction with, and physical
-proximity to, their victim for the remainder of the event
-* early termination of a talk that violates the policy
-* not publishing the video or slides of a talk that violated the policy
-* not allowing a speaker who violated the policy to give (further) talks at
-the event
+* warning the offender to cease their behavior, letting them know that
```
Comment (mjgiarlo) [inline code_of_conduct.md:104]: Changing `harasser` to `offender` throughout this section for consistency with the text in the rest of the doc above.

File: code_of_conduct.md (hunk 6 above)
Comment (phette23) [reply code_of_conduct.md:36]: I agree, better diction.

**Review by phette23 — APPROVED** [review]
These look great to me. Given that these edits are mostly about clarity, grammar, and some word choices that are improvements but not substantial changes to the sense of various passages, I don't think we have to run this through a community vote to merge.

**Pull request merged**